draft-ietf-quic-transport.md

```diff
@@ -1645,246 +1645,190 @@ to get blocked.
 
 # Error Codes {#error-handling}
 
-This section lists all the QUIC error codes that may be used in a
-CONNECTION_CLOSE frame.  TODO: Trim list and group errors for readabiity.
+Error codes are 32 bits long, with the first two bits indicating the source of
+the error code:
 
-TODO: Discuss error handling beyond just listing error codes.
+0x0000-0x3FFF:
```

Comment:
Oh, and to that end, maybe also move the host local to the same space, so we have: app, host, quic, security.